From 02de641b799dc4d6eb96335e698de0cdb9889549 Mon Sep 17 00:00:00 2001
From: Igor Opaniuk <igor.opaniuk@foundries.io>
Date: Mon, 22 Feb 2021 23:08:25 +0200
Subject: [PATCH 090/322] [FIO internal] imx8mqevk: drop hardcoded USB
 configuration

Drop hardcoded forced USB configuration from imx8mqevk header, so we can
adjust/disable it in defconfig.

Signed-off-by: Igor Opaniuk <igor.opaniuk@foundries.io>
Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 include/configs/imx8mq_evk.h | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/include/configs/imx8mq_evk.h b/include/configs/imx8mq_evk.h
index 68e5898aefd..6e2c93b042e 100644
--- a/include/configs/imx8mq_evk.h
+++ b/include/configs/imx8mq_evk.h
@@ -203,17 +203,6 @@
 /* I2C Configs */
 #define CONFIG_SYS_I2C_SPEED		  100000
 
-/* USB configs */
-#ifndef CONFIG_SPL_BUILD
-
-#define CONFIG_CMD_USB_MASS_STORAGE
-#define CONFIG_USB_GADGET_MASS_STORAGE
-#define CONFIG_USB_FUNCTION_MASS_STORAGE
-
-#define CONFIG_CMD_READ
-
-#endif
-
 #define CONFIG_SERIAL_TAG
 #define CONFIG_FASTBOOT_USB_DEV 0
 
-- 
2.34.1

